components/ModDashboardLeftNavbar.vue

```diff
@@ -36,7 +36,8 @@
 </template>
 
 <script setup lang="ts">
-import { ref, Ref } from 'vue'
+import { ref} from 'vue'
+import type { Ref } from 'vue'
```

Comment:
you can keep these in the same line, just put `type` in front of `Ref`  within the curlies